api/include/opentelemetry/trace/trace_flags.h

```diff
@@ -19,13 +19,30 @@ class TraceFlags final
 {
 public:
   static constexpr uint8_t kIsSampled = 1;
+  static constexpr uint8_t kIsRandom  = 2;
+
+  /**
+   * Valid flags in W3C Trace Context version 1.
+   * kIsSampled.
+   * See https://www.w3.org/TR/trace-context-1/#trace-flags
+   */
+  static constexpr uint8_t kAllW3CTraceContext1Flags = 0x01;
+
+  /**
+   * Valid flags in W3C Trace Context version 2.
+   * kIsSampled, kIsRandom.
```

Comment:
nit: instead of add this in the comments, how about defined the flags as `kIsSampled | kIsRandom`?

Reply:
Thanks, fixed.